HW_7/task_34.py

```diff
@@ -0,0 +1,25 @@
+# Задача 34: Винни-Пух попросил Вас посмотреть, есть ли в его стихах ритм. 
+# Поскольку разобраться в его кричалках не настолько просто, насколько легко он их придумывает, Вам стоит написать программу. 
+# Винни-Пух считает, что ритм есть, если число слогов (т.е. число гласных букв) в каждой фразе стихотворения одинаковое. 
+# Фраза может состоять из одного слова, если во фразе несколько слов, то они разделяются дефисами. 
+# Фразы отделяются друг от друга пробелами. Стихотворение Винни-Пух вбивает в программу с клавиатуры. 
+# В ответе напишите “Парам пам-пам”, если с ритмом все в порядке и “Пам парам”, если с ритмом все не в порядке.
+
+# Ввод: 
+# пара-ра-рам рам-пам-папам па-ра-па-дам 
+
+# Вывод:
+# Парам пам-пам
+
+
+def f(word):
+    return sum(1 for i in word if i in 'аеёиоуыэюя')
+    
+    
+word = input('Введите фразу или слово для определения ритма: ')
+str = word.lower().split()
+temp = f(str[0])
+if all([f(i) == temp for i in str]):
```

Comment:
Функция `all` умеет работать с простым генератором
```suggestion
if all(f(i) == temp for i in str):
```

Reply:
понял, спасибо!